SofaKernel/modules/SofaCore/src/sofa/core/ExecParams.h

```diff
@@ -133,16 +133,16 @@ class SOFA_CORE_API ExecParams
     }
 
     ////////////////////////////////////// DEPRECATED ///////////////////////////////////////////
-    SOFA_DEPRECATE_ASPECT("Aspects have been removed, you can safely remove the function from your code.")
+    SOFA_ATTRIBUTE_DISABLED("2020-03-25 PR#1269", "2021-02-08 PR#1793", "Aspects have been removed, you can safely remove the function from your code.")
```

Comment:
There should be a macro SOFA_ATTRIBUTE_DISABLED_ASPECT instead of using a generic macro.